projects/ad719x/builds.json

```diff
@@ -0,0 +1,10 @@
+{
+      "xilinx": {
+            "dummy_example": {
+                  "flags": "DUMMY_EXAMPLE=y",
+                  "hardware": [
+                        "ad719x_xilinx_zed"
```

Comment:
Is this a valid HDL project (https://github.com/analogdevicesinc/hdl/tree/master/projects) ?

Reply:
The .xsa file which was used is a generic project with one SPI channel and GPIO pins on a PMOD connector. It was used on multiple drivers I've developed.

Reply:
OK, let's specify for the moment an available project, https://github.com/analogdevicesinc/hdl/tree/master/projects/adv7511/zed for example.